src/Table.cpp

```diff
@@ -0,0 +1,100 @@
+//
+// Created by derekjtong on 4/17/2023.
+//
+#include "sudoManager/Table.hpp"
+
```

Comment:
```suggestion
#include "sudoManager/Table.hpp"
#include <stdexcept>
```
@derekjtong 
This will fix the **Ubuntu-20-04-x86_64** check